run-tests.py

```diff
@@ -27,8 +27,9 @@
 # POSSIBILITY OF SUCH DAMAGE.
 #
 
+from __future__ import print_function
```

Comment:
Not strictly necessary since this is mostly used for consistent STDERR printing when we could just use STDOUT -- it's just the test file name.

Reply:
🤔 Minor note here but i believe the main change to the print function between 2 and 3 was that in python 3 you [are _forced_ to use the call syntax][1] `print("stuff")` instead of the original python statement syntax `print "stuff"`

The `print` function was backported to at least Python 2.7, so you shouldn't have to import anything just for that.

 [1]: https://docs.python.org/3/whatsnew/3.0.html#print-is-a-function

Reply:
Lol nm it's [printing a tuple][1] in 2.7. Could have sworn 🤔 but must have always involved `from __future__`

  [1]: https://stackoverflow.com/a/33996938/832136

Reply:
I had the same confusion when I first his this after running `2to3` -- sort of makes sense with 2's ability to just:

```python
print some, "arbitrary", comma, { 'separated': 'stuff' }
```

`six` has `_print` for this but didn't want to dig further into `pdagent` than we need to considering this is just supposed to be a test runner.